tests/shared.rs

```diff
@@ -141,11 +142,11 @@ fn massage_windows_line(line: &str) -> String {
 }
 
 fn strip_trailing_carriage_return(line: &str) -> Cow<'_, str> {
-    regex!(r#"\r$"#).replace(line, "")
+    regex!(r#"\r((?:\u{1b}\[\d+m)*)$"#).replace(line, "$1")
 }
 
 fn normalize_match_path(line: &str) -> Cow<'_, str> {
-    regex!(r#"^[^:]+[:-]\d+[:-]"#)
+    regex!(r#"^(?:\u{1b}\[\d+m)*[a-zA-Z_\-\\/]+\.(?:rs|js|tsx)"#)
```

Comment:
And here this regex for finding "paths" (for changing `\` -> `/` on Windows) keeps getting re-worked, now it really can't really on "context" at all (since eg heading lines are just paths by themselves)

So just expressed it in a way that works for all current tests with filename output